x-pack/plugins/security_solution/public/resolver/models/process_event.ts

```diff
@@ -98,45 +92,32 @@ export function uniquePidForProcess(passedEvent: ResolverEvent): string {
 /**
  * Returns the PID for the process on the host
```

Comment:
:grey_question: Given how often the question comes up, it might be nice at some point to include in the doc comments some descriptions of e.g. when these things are unique and when they aren't in various scenarios.

Reply:
Sounds good. I'm also trying to do away with the uniquePid nomenclature.